pkg/api/networkservice/connectioncontext.proto

```diff
@@ -39,9 +39,9 @@ message IpFamily {
 }
 
 message ExtraPrefixRequest {
-    IpFamily addr_family    = 1;
-    uint32 prefix_len       = 2;
-    uint32 required_number  = 3;
+    IpFamily addr_family = 1;
```

Comment:
Again with the whitespace changes... do we need to settle on a linter so we can come to one agreed set of whitespace?